CMSgov/onemac
# Oy2 21129 - new kafka streams into onemac

Story: https://qmacbis.atlassian.net/browse/OY2-21129
Endpoint: See github-actions bot comment

### Details

Create new stream service for new topics. Ensure topic name change can restart topic without duplicating records.

### Changes

- new service for topics: seatool-sink
- turn off old service: stream-functions
- empty stream function for data inserts (future work)

### Test Plan

1. Verify inserted records have all appropriate data in pk sk gsi1pk gsi1sk
2. Verify changing name of topic will reset streams
